apps/fluent-tester/src/TestComponents/Chip/E2EChipTest.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { View } from 'react-native';
+
+import { Chip } from '@fluentui-react-native/chip';
+import type { InteractionEvent } from '@fluentui-react-native/interactive-hooks';
+import { Text } from '@fluentui-react-native/text';
+
+import { CHIP_TEST_COMPONENT, CHIP_END_TEXT, CHIP_START_TEXT, CHIP_TEXT } from '../../../../E2E/src/Chip/consts';
+import { testProps } from '../Common/TestProps';
+
+export const E2EChipTest: React.FunctionComponent = () => {
+  const [text, setText] = React.useState<string>(CHIP_START_TEXT);
+  const showTextOnSelection = React.useCallback(
+    (_e: InteractionEvent, isSelected: boolean) => setText(isSelected ? CHIP_END_TEXT : CHIP_START_TEXT),
```

Comment:
Let's improve the naming of `CHIP_END_TEXT` and `CHIP_START_TEXT`. These names don't really describe the intent behind the usage.

Reply:
Renamed